pixeltable/metadata/converters/convert_28.py

```diff
@@ -0,0 +1,15 @@
+import logging
+
+import sqlalchemy as sql
+
+from pixeltable.metadata import register_converter
+from pixeltable.metadata.schema import Dir, Table, TableSchemaVersion, TableVersion
+
+
+@register_converter(version=28)
+def _(engine: sql.engine.Engine) -> None:
+    with engine.begin() as conn:
+        conn.execute(sql.update(Dir).values(md=Dir.md.concat({'user': None, 'additional_md': {}})))
```

Comment:
Do we need to put 'user': None in there?

Reply:
Yes. Dataclass deserialization expects it to be present in the dict.